datahub-web-react/src/app/ingest/secret/SecretBuilderModal.tsx

```diff
@@ -47,6 +47,7 @@ export const SecretBuilderModal = ({ initialState, visible, onSubmit, onCancel }
             title={<Typography.Text>Create a new Secret</Typography.Text>}
             visible={visible}
             onCancel={onCancel}
+            zIndex={1051} // one higher than other modals - needed for managed ingestion forms
```

Comment:
should we store these as constant and constant+1 so if one is changed this modal doesn't disappear for (seemingly) no reason?

Reply:
ugh i wanted to do that, but it's Ant designs constant of 1050 so we don't actually have access to that in our code base :( so this is risky if they change that number for some reason. we could just plop this at some ridiculously high number as well if we really wanted to